### PR TITLE
Added schemaOnly and AWS credentials support

### PR DESCRIPTION
**Pull request by seliger**
The --schemaOnly option allows the user to backup and restore
just the schema information for the given table(s). The use
case for this is if you need to migrate your tables from one
AWS account to a different AWS account.

(Our specific use case: We have an AWS account for development
and yet a separate AWS account for production.)

To support the use case of multiple AWS accounts, -p or --profile
was also added so that the user could explicitly specify which
profile to use from the AWS credentials file (~/.aws/credentials).
This is /slightly/ more secure than putting accessKey and
secretKey into the command line.

### CODE REVIEW

**Comment by bchew** [comment]
@seliger apologies for the delay in getting to your PR, could you resolve the conflicts before I review it again, thanks!

**Comment by seliger** [comment]
@bchew, that should hopefully do it. If I need to do anything else, let me know.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 dynamodump
 ==========
 
-[![Buildstatus](https://travis-ci.org/bchew/dynamodump.svg)](https://travis-ci.org/bchew/dynamodump)
```
Comment (bchew) [inline README.md]: Can you revert this change

**Comment by bchew** [comment]
@seliger just a change which needs to be re-added back, and should be good to go, thanks!

**Comment by seliger** [comment]
@bchew Sorry for the delay. This should now merge cleanly. Nice use of CI for testing. FYI, I built a similar tool for extracting IAM roles, largely based off of your concept. Thank you for making this! It has been very useful.

**Comment by bchew** [comment]
@seliger merged, thanks for making the changes! Have made some minor help changes after and just synced the readme with it https://github.com/bchew/dynamodump/commit/20979c4cc815cd31c6b90e0fce774ad19ea60af4

Thanks for letting me know about what you built, and glad you've found this tool useful :)
